app-backend/database/src/main/scala/com/azavea/rf/database/tables/Exports.scala

```diff
@@ -0,0 +1,228 @@
+package com.azavea.rf.database.tables
+
+import com.azavea.rf.database.{Database => DB}
+import com.azavea.rf.database.ExtendedPostgresDriver.api._
+import com.azavea.rf.database.fields.{OrganizationFkFields, TimestampFields}
+import com.azavea.rf.database.query.{ExportQueryParameters, ListQueryResult}
+import com.azavea.rf.datamodel._
+
+import com.lonelyplanet.akka.http.extensions.PageRequest
+import com.typesafe.scalalogging.LazyLogging
+import slick.model.ForeignKeyAction
+import io.circe.Json
+import cats.implicits._
+
+import java.sql.Timestamp
+import java.util.UUID
+import java.net.URI
+
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+
+/** Table that represents exports
+  *
+  * Exports represent asynchronous export tasks to export data
+  */
+class Exports(_tableTag: Tag) extends Table[Export](_tableTag, "exports")
+  with TimestampFields
+  with OrganizationFkFields
+{
+  def * = (id, createdAt, createdBy, modifiedAt, modifiedBy, organizationId, projectId, sceneIds, exportStatus,
+    exportType, visibility, exportOptions) <> (
+    Export.tupled, Export.unapply
+  )
+
+  val id: Rep[java.util.UUID] = column[java.util.UUID]("id", O.PrimaryKey)
+  val createdAt: Rep[java.sql.Timestamp] = column[java.sql.Timestamp]("created_at")
+  val createdBy: Rep[String] = column[String]("created_by", O.Length(255,varying=true))
+  val modifiedAt: Rep[java.sql.Timestamp] = column[java.sql.Timestamp]("modified_at")
+  val modifiedBy: Rep[String] = column[String]("modified_by", O.Length(255,varying=true))
+  val organizationId: Rep[java.util.UUID] = column[java.util.UUID]("organization_id")
+  val projectId: Rep[java.util.UUID] = column[java.util.UUID]("project_id", O.PrimaryKey)
+  val sceneIds: Rep[List[java.util.UUID]] = column[List[java.util.UUID]]("scene_ids")
```

Comment:
Is this necessary if we have a project? Though this would allow for users to export a subset of scenes from a project which would be good. 

It could be onerous though if a project has many scenes -- might be something we can deal with later though if that becomes a problem.